debug.log

```diff
@@ -0,0 +1,13 @@
+[0104/112632.596:ERROR:directory_reader_win.cc(43)] FindFirstFile: The system cannot find the path specified. (0x3)
```

Comment:
@jazznerd206 Is this file required to be here or is it more like a cypress out put thing? If it isn't required can we have dubug.log put into gitignore

Reply:
The debug file can go into the gitignore, I'll put it in there for the following commit.